### DISCUSSION TITLE
Add way to change the featured artist default

### DISCUSSION THREAD

**Discussion by JeremyMoeglich** [ppy/osu#22107] [Ideas] [answered]
I'm fine with featured artists becoming a default in #21646, but there should be a way to change that default even if it's hidden somewhere in the settings.
For me toggling this setting is basically necessary every time I search
#22027 also references this as a minor issue

More in general, changing search filter defaults would be nice, maybe using a save filters button

**Comment by peppy** [comment, marked as answer]
#22009

**Reply by JeremyMoeglich** [reply]
I know that you can change the filter, I'm looking for a way to preserve the filter even when restarting the game.
Sorry if the previous text was unclear

**Reply by peppy** [reply]
Yep, understood.

**Reply by darkralts** [reply]
@peppy can you be more clear with your intentions as to whether you would like to add this feature?

**Reply by darkralts** [reply]
i feel the same way as jeremy, that it is kind of counter user friendly to have to constantly keep turning this setting off.
